web-ui/src/components/certifications/Certifications.jsx

```diff
@@ -0,0 +1,254 @@
+import PropTypes from 'prop-types';
+import React, { useCallback, useEffect, useState } from 'react';
+
+import {
+  Button,
+  Dialog,
+  DialogContent,
+  DialogTitle,
+  TextField
+} from '@mui/material';
+import Autocomplete, { createFilterOptions } from '@mui/material/Autocomplete';
+
+import ConfirmationDialog from '../dialogs/ConfirmationDialog';
+
+import './Certifications.css';
+
+const certificationBaseUrl = 'http://localhost:3000/certification';
```

Comment:
Currently this will only work for me because it relies on endpoints that I implemented in a mock server that I run locally. I need to work with a Micronaut developer to get them ported over.